### ISSUE TITLE
requirements.txt == versus >=

### ISSUE THREAD

**Issue by dukedougal** [mattupstate/overholt#25]
I'm not sure about this but should the requirements.txt contain == or >=

If == does it install exactly that version of the requirement rather than >= installing the most recent version of the requirement?

**Comment by mattupstate** [comment]
Yes, that is correct. It depends on how strict you'd like your dependencies to be.
